README.md

```diff
@@ -307,6 +307,49 @@ class App extends Component {
   }
 }
 ```
+##### React Using Hooks
+
+```js
+import React,{ useState, useEffect } from 'react';
+import ApolloClient, { InMemoryCache } from "apollo-boost";
```

Comment:
This library is no longer recomended.

Reply:
have had created another PR for it and corrected it.
TNX for correcting me :) as you know this was my first PR to a major repo and it really feels great :)